### ISSUE TITLE
Like default theme use get_theme_file_uri function instead of get_template_directory_uri

### ISSUE THREAD

**Issue by vinoddalvi** [Automattic/_s#1141]
Use newly introduced function [get_theme_file_uri](https://developer.wordpress.org/reference/functions/get_theme_file_uri/) that makes it easy to overwrite files in child theme. The Twenty Seventeen theme already uses this function.

**Comment by m-e-h** [comment]
Wouldn't `get_parent_theme_file_uri` be more appropriate here? 
If there's a child theme, `get_theme_file_uri` would first look there for `/js/customizer.js` and use that instead. I could see this causing problems. 
I don't see a child theme wanting to override these scripts unless it's unintentional.

**Comment by samikeijonen** [comment]
Before we started using new functions 2 major version back if I remember correctly. That means that those functions can be used in version 4.9 because they arrived in 4.7.

**Comment by davidakennedy** [comment]
I dropped some similar comments to @samikeijonen on the PR since I saw that first.

**Comment by samikeijonen** [comment]
We can proceed this now.

**Comment by davidakennedy** [comment]
I second that we can proceed here. PRs welcome, if one of the maintainers doesn't get to it first.

**Comment by jrfnl** [comment]
Note for whoever creates the PR for this:
Please don't forget to update the `Requires at least`  WP version in the `readme.txt` and the same in the `phpcs.xml.dist` file.

**Comment by m-e-h** [comment]
To clarify, is `get_parent_theme_file_uri` what we want here rather than `get_theme_file_uri`?

**Comment by samikeijonen** [comment]
`get_theme_file_uri` for assets because the idea is that child themes can actually overwrite assets files. And perhaps `get_parent_theme_file_path` for requiring files in `inc` folder at the same time.

**Comment by m-e-h** [comment]
I was actually thinking the opposite. 😯

Asset files can easily be `dequeued`. 
Like I mentioned above, I have several child themes with a `/js/customizer.js` or something like a `/js/main.js` file in them. I wouldn't want or expect that to override the the parent themes js.

**Comment by samikeijonen** [comment]
That's only my opinion because for novice site owners that's the easier way, talking only about public themes here.

**Comment by wpexplorer** [comment]
I agree with m-e-h using get_parent_theme_file_path and get_parent_theme_file_uri would be the ideal functions to use. Many people like to use the same names for scripts/files in their child themes (no matter what the parent theme is) without having to look into the parent to see if it will create a conflict.
